documentation/API_documentation/customer-insights-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,28 @@
+# API Readiness Checklist
+
+Checklist for Customer Insights v0.1.0-rc.1 in r1.1
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | [link](/code/API_definitions/customer-insights.yaml) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y   | [r2.2](https://github.com/camaraproject/Commonalities/releases/tag/r2.2) |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   | [r2.1](https://github.com/camaraproject/IdentityAndConsentManagement/releases/tag/r2.1) |
```

Comment:
You can consider to mention here r2.2 as well already (except you are seeing changes which are relevant for your API and not yet applied).

```suggestion
|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   | [r2.1](https://github.com/camaraproject/IdentityAndConsentManagement/releases/tag/r2.1) |
```